.github/workflows/main.yml

```diff
@@ -0,0 +1,80 @@
+name: 'build and test'
+on: [push]
+
+jobs:
+  build-wrappers-macos:
+    runs-on: macos-latest
+    name: 'Build macOs wrappers'
+    steps:
+    - name: 'Checkout code'
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    # - name: 'Check if cache contains wrappers already'
+    #   id: wrapper-cache
+    #   uses: actions/cache@v2
+    #   with:
+    #     path: ./wrappers/build/
+    #     key: wrapper-macos-${{hashfiles('./wrappers/**')}}
+    - name: 'Build wrappers'
+      # if: 'steps.wrapper-cache.outputs.cache-hit != true'
+      uses: ./.github/actions/build_wrappers
+      env: 
+        REALM_CMAKE_CONFIGURATION: Release
+      with:
+        cmds: '[ { "cmd": "./wrappers/build-macos.sh", "cmdParams": [] } ]'
```

Comment:
I understand this makes possible to use multiple tuples of (command, parameters), but it looks a little bit clunky to use json here. I suppose in most cases parameters will be embedded in commands already, so I am not sure it's necessary, but that's up for discussion.

Another thing to consider is that in most cases we will have only one command, so it would be nice if the action would just accept a string as an input, without the need to embed it in json.

Reply:
I am super open for discussion as I am not sure what the best solution would be about that. However, the reason for commands and their parameters to be diveded instead of being 1 string is the fact that I decided to launch command line command with `child_process.spawn` instead of `child_process.exec`. The latter accepts a string with all in it, the former, as far as I could find in the documentation, wants the arguments passed as list. So, I temporarily took the shortcut of having the JSON library to parse those for me. We can reshape all of that when we've agreed on the final solution.